package.json

```diff
@@ -13,7 +13,7 @@
   "main": "./electron/lib/index.prod.js",
   "scripts": {
     "precommit": "lint-staged",
-    "lint:fix": "tslint --fix '{src,stories}/**/*.{ts,tsx}'",
+    "lint:fix": "tslint --fix '{src,stories}/**/*.{ts,tsx}' && eslint '{src,stories,electron/src}/**/*.{ts,tsx}' --quiet --fix --no-eslintrc -c ./.eslintrc.json",
```

Comment:
I never used `lint:fix`, I think. Was just part of some boilerplate of ours if I'm not mistaken.

**What would be way more important is to actually lint the code 😉**

Please adapt the `test` script.